Sources/XCRemoteCache/Config/XCRemoteCacheConfig.swift

```diff
@@ -27,7 +27,7 @@ public enum XCRemoteCacheConfigError: Error {
 
 public struct XCRemoteCacheConfig: Encodable {
     /// Remote cache schema version. Bump that version if RC artifact generation introduces breaking changes
-    let schemaVersion = "5"
+    let schemaVersion = "6"
```

Comment:
No need to bump schema: `CLANG_COVERAGE_MAPPING`, similar to `CLANG_PROFILE_DATA_DIRECTORY` is not defined for a build with coverage disabled. For coverage enabled, new value (`YES`) will invalidate previous artifacts anyway (`CLANG_PROFILE_DATA_DIRECTORY` is an absolute path).

```suggestion
    let schemaVersion = "5"
```